app/pulp/app/models/task.py

```diff
@@ -170,8 +147,8 @@ def set_completed(self, result):
 
         This updates the :attr: `finished_at` and sets the :attr: `state` to :attr: `COMPLETED`.
 
-        :param result: The result to save on the :class: `~pulp.app.models.Task`
-        :type result: dict
+        Args:
+            result (dict): The result to save on the :class: `~pulp.app.models.Task`
```

Comment:
There should be no space between  `:class:` and `~pulp.app.models.Task`, otherwise docs are not generated correctly.